lib/MusicBrainz/Server/Edit/Series/Edit.pm

```diff
@@ -82,6 +89,8 @@ sub build_display_data {
 around allow_auto_edit => sub {
     my ($orig, $self, @args) = @_;
 
+    return 1 if $self->can_amend($self->series_id);
```

Comment:
Yeah, sorry, I realized after I wrote that that whether or not it's an auto-edit shouldn't affect that.